susemanager-utils/testing/automation/schema-migration-test-pgsql.sh

```diff
@@ -4,6 +4,23 @@ HERE=`dirname $0`
 . $HERE/VERSION
 GITROOT=`readlink -f $HERE/../../../`
 
+# File created by Gitarro with info about a PR, it only exists when we are testing a PR
+GITARRO_JSON="${GITROOT}/.gitarro_pr.json"
+
+# Base version for the idempotency test. All PostgreSQL since this version must be idempotent for the test to pass
+# This value is is used if the file for the PR is not found (we assume we are on a branch)
+BASE_VERSION='4.0.1'
```

Comment:
we might put the base_version in the https://github.com/uyuni-project/uyuni/blob/1c0c04d036b7c2693585140c8a49d388533ab2c8/susemanager-utils/testing/automation/VERSION file?

From external POV we might call this var `BASE_VERSION` like `PGSQL_BASE_VERSION` if it has impact for base_version.

Reply:
Fixed and testing.